messages/de/app.php

```diff
@@ -191,16 +191,16 @@
     '{0, plural, =1{# hour} other{# hours}} ago' => '{0, plural, =1{# Stunde} other{# Stunden}}',
     '{0, plural, =1{# minute} other{# minutes}} ago' => '{0, plural, =1{# Minute} other{# Minuten}}',
     '{0, plural, =1{# second} other{# seconds}} ago' => '{0, plural, =1{# Sekunde} other {# Sekunden}}',
-    'DISBOARD was made to make everyone enjoy Discord more.<br>We hope you find awesome Discord servers and friends here 😊.' => 'DISBOARD ist gemacht worden, damit alle Discord mehr genießen können.<br>Wir hoffen, du findest tolle Discord Server und Freunde hier 😊.',
+    'DISBOARD was made to make everyone enjoy Discord more.<br>We hope you find awesome Discord servers and friends here 😊.' => 'DISBOARD ist gemacht worden, damit alle Discord mehr genießen können.<br>Wir hoffen, du findest hier tolle Discord Server und Freunde 😊.',
     /* Rules, ToS */
-    'Name or description contains insult or swear words' => 'Der Name oder die Beschreibung enthalten Beleidigungen oder Flüche',
+    'Name or description contains insult or swear words' => 'Der Name oder die Beschreibung enthält Beleidigungen oder Fluchwörter',
```

Comment:
“enthalten” would be correct here, specifically citing the [Duden](https://www.duden.de/sprachwissen/sprachratgeber/%E2%80%9Esowohl-als-auch%E2%80%9C-%E2%80%93-Singular-oder-Plural-Auf%C2%ADz%C3%A4hlung%C2%ADen) again:
> So wird der Singular verwendet, wenn das Verb vor dem Subjekt steht und
>
> 1. zwei beliebige Substantive ohne Konjunktion miteinander verbunden sind („Schon kam der Junge, der Hund angelaufen“),
> 2. die Subjektteile zwar durch eine Konjunktion verbunden, aber durch das Verb getrennt sind („Dieses Buch gefällt mir sowie das blaue dort drüben“) oder
> 3. eine Aufzählung in Tabellenform eingeleitet wird („Benötigt wird: ein Hammer, ein Zollstock, eine Wasserwaage …“).


Changing to “Fluchwörter” is alright.

Reply:
Der Name (sg)  **oder** die Beschreibung (sg) :   ist entweder Name oder Beschreibung nicht beides, daher ist enthalten singular

Reply:
Thanks for clarifying. I looked it up again and understand this as well now. Sorry for the confusion. 😅